server/src/mcr/activity-data/activityData.spec.js

```diff
@@ -44,6 +47,7 @@ describe(`${typeName} mongoose schema testing`, function () {
         done()
       })
       .catch(err => {
+        console.log('error >> ', err)
```

Comment:
use debug and it is even more important that error messages  can be located.  Use appropriate prefix in the message.